src/nvapi_disp.cpp

```diff
@@ -195,8 +195,8 @@ extern "C" {
                 pHDRColorDataV2->hdrDynamicRange = NV_DYNAMIC_RANGE_VESA;
                 pHDRColorDataV2->hdrBpc = data.BitsPerColor;
             } else {
+                // Ignoring some extended properties of HDRColorDataV2.
```

Comment:
Actually, I liked the log message because we are explicitly and by design not doing what is asked of us. We could always make it print no more than once if it was too spammy.

Reply:
Seeing this log statement in the logs, my thoughts were that it didn't add much and might just be confusing for users, also there is no way to "make it go away" for a user or application since it will always appear for V2 and Set (which i guess will be 90% of all usages of that method).  
But i see your point too, we can reconsider in the future.